com.github.robertsanseries.ciano.json

```diff
@@ -54,24 +56,50 @@
             "--without-xml",
             "--without-zlib"
           ],
-          "sources": [
-            {
-              "type": "archive",
-              "url": "https://github.com/ImageMagick/ImageMagick/archive/7.0.11-6.tar.gz",
-              "sha256": "8adc1605784653b078572b825e8cd1d3d54f8a1b4ba86b32ca253c038f7e4c37"
-            }
-          ]
+          "sources": [{
+              "type": "git",
+              "url": "https://github.com/ImageMagick/ImageMagick.git",
+              "tag": "7.1.1-38",
+              "commit": "b0ab92265bab638e6ecd2f18b45977c38771c671"
+          }]
         },
         {
             "name": "chrpath",
             "cleanup": ["*"],
-            "sources": [
-                {
-                  "type": "archive",
-                  "url": "http://deb.debian.org/debian/pool/main/c/chrpath/chrpath_0.16.orig.tar.gz",
-                  "sha256": "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
-                }
-            ]
+            "sources": [{
+                "type": "archive",
+                "url": "http://deb.debian.org/debian/pool/main/c/chrpath/chrpath_0.16.orig.tar.gz",
+                "sha256": "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
+            }]
```

Comment:
these formatting change are what I was talking about. The obscure real change and brings nothing as they don't even follow the usual formatting.